src/shadowbox/server/server_access_key.spec.ts

```diff
@@ -220,22 +302,22 @@ describe('ServerAccessKeyRepository', () => {
     done();
   });
 
-  it('can remove access key limits', async (done) => {
+  it('can remove the default data limit', async (done) => {
     const limit = {bytes: 100};
-    const repo = new RepoBuilder().accessKeyDataLimit(limit).build();
+    const repo = new RepoBuilder().defaultDataLimit(limit).build();
     expect(repo.dataLimit).toEqual(limit);
-    await expectNoAsyncThrow(repo.removeAccessKeyDataLimit.bind(repo));
+    await expectNoAsyncThrow(repo.removeDefaultDataLimit.bind(repo));
     expect(repo.dataLimit).toBeUndefined();
     done();
   });
 
-  it('removeAccessKeyDataLimit restores over-limit access keys', async (done) => {
+  it('removeDefaultDataLimit restores over-limit access keys', async (done) => {
```

Comment:
I think it may be useful to enhance the limit enforcement test to make sure we are enforcing per-key limits.

Reply:
Done in 20ad19e